tests/integration/components/frost-bunsen-form/renderers/datetime-test.js

```diff
@@ -45,10 +46,15 @@ describe('Integration: Component / frost-bunsen-form / renderer / datetime', fun
   })
 
   it('renders as expected', function () {
-    expectCollapsibleHandles(0)
     expectBunsenDatetimeRendererWithState('foo', {label: 'Foo'})
     expectOnValidationState(ctx, {count: 1})
   })
+  it('should have an input for date', function () {
+    expect($hook('bunsenForm-foo-datetimePicker-date-input').length).to.equal(1)
```

Comment:
I don't like seeing these expects in here. These should be covered under the test helpers for your custom renderer.